.travis.yml

```diff
@@ -17,6 +18,6 @@ matrix:
 
 before_script:
     - composer self-update
-    - composer install --no-interaction --prefer-source --dev
```

Comment:
The `--dev` option is deprecated.

The deprecated warning message is as follows:

```
You are using the deprecated option "dev". Dev packages are installed by default now.
```